Makefile

```diff
@@ -36,6 +38,10 @@ verify: ## Verify code. Includes dependencies, linting, formatting, etc
 	go fmt ./...
 	golangci-lint run
 
+verify-licenses: ## Verifies dependency licenses and requires GITHUB_TOKEN to be set
```

Comment:
consider just `make licenses`

Reply:
👍  I suppose you get more details in `make help` anyways